api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorSpec.java

```diff
@@ -32,6 +32,7 @@
 
     private String className;
     private Integer tasksMax;
+    private Boolean pause;
```

Comment:
Pausing a connector stops its tasks without causing a rebalance. So it's less invasive to the cluster as a whole than deleting the connector and recreating it later. One use case would be, for example, pausing a connector while performing some upgrade on the external system (to avoid generating lots of errors).

Reply:
So I guess it is across multiple reconciliations and for more than few minutes. I guess it makes sense to have it as field than. As annotation, it would be more as `resume` annotation and `pause` annotation and that would cause mess if we later start handling failed connectors etc. So I guess we can keep is more like this to describe a state and not an action.